aesm-client/src/lib.rs

```diff
@@ -179,12 +179,16 @@ impl AesmClient {
         report: Vec<u8>,
         spid: Vec<u8>,
         sig_rl: Vec<u8>,
+        quote_type: QuoteType,
+        nonce: [u8; 16],
```

Comment:
The other parameters are `Vec<u8>`, even though they have a fixed size. Probably best to be consistent.

Reply:
ok